generators/app/index.js

```diff
@@ -29,8 +31,14 @@ CrowdtapAngularGenerator.prototype.getAppName = function() {
     {
       name: 'appPort',
       message: "Enter the port number your app will run on locally:"
+    },
+    {
+      name: 'sentrySetup',
+      message: ["Please visit https://crowdtap.atlassian.net/wiki/display/EN/2014/10/16/Sentry+Part+I%3A+Migration",
+      "follow the instructions under 'Front end apps' and 'integrations'",
+      "enter 'yes' if you have set up Sentry for your new app:"].join('\n')
```

Comment:
I thought of that, but we dont really do anything with the prompt value. They need to update assets.shared and crowdtap.